app/assets/v2/js/pages/bounty_details.js

```diff
@@ -496,25 +496,52 @@ var show_interest_modal = function() {
         modalClass: 'modal add-interest-modal'
       });
 
+      var placeholder = gettext(
+        'What steps will you take to complete this task?\n\n' +
+        'Example:\n' +
+        'I\'d start by implementing Netflify, then I\'d make mockups ' +
+        'for the /blog page and individual post pages and proceed ' +
+        'with building them after getting some feedback from the team.' +
+        'I will start work on this right away and have it ready by the end of the weekend.');
+
+      var textarea = modal.find('textarea');
+
+      textarea.addClass('placeholder');
+      textarea.val(placeholder);
+
+      textarea.on('focus', function(event) {
+        if (textarea.val() === placeholder) {
+          textarea.val('');
+          textarea.removeClass('placeholder');
+        }
+      });
+
+      textarea.on('blur', function(event) {
+        if (!event.target.value) {
+          event.target.value = placeholder;
+          textarea.val(placeholder);
+          textarea.addClass('placeholder');
+        }
+      });
+
       modal.on('submit', function(event) {
         event.preventDefault();
 
         var issue_message = event.target[0].value.trim();
-        var agree_precedence = event.target[1].checked;
-        var agree_not_to_abandon = event.target[2].checked;
 
-        if (!issue_message) {
-          _alert({message: gettext('Please provide an action plan for this ticket.')}, 'error');
+        if (!issue_message || issue_message === placeholder || issue_message.length < 48) {
+          _alert({message: gettext('Please provide an action plan for this ticket. At least two sentences. (48 Characters)')}, 'error');
           return false;
         }
 
-        if (!agree_precedence) {
-          _alert({message: gettext('You must agree to the precedence clause.')}, 'error');
-          return false;
-        }
-        if (!agree_not_to_abandon) {
-          _alert({message: gettext('You must agree to keep the funder updated on your progress.')}, 'error');
-          return false;
+        var checkboxes = modal.find('input[type=checkbox]');
+        var len = checkboxes.length;
+
+        while (len-- > 0) {
+          if (!checkboxes[len].checked) {
```

Comment:
Again, a pretty generic validation which I am not very happy about thinking about future, when it might so happen that we have checkboxes which don't necessarily require to be checked and is an optional choice.

cc: @PixelantDesign @mbeacom